plugin.json

```diff
@@ -0,0 +1,88 @@
+{
+    "id": "jitsi",
+    "name": "Jitsi",
+    "description": "Jitsi audio and video conferencing plugin for Mattermost.",
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-jitsi",
+    "support_url": "https://github.com/mattermost/mattermost-plugin-jitsi/issues",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-jitsi/releases/tag/v1.3.0",
+    "icon_path": "assets/icon.svg",
+    "version": "1.3.0",
+    "min_server_version": "5.2.0",
+    "server": {
+        "executables": {
+            "linux-amd64": "server/dist/plugin-linux-amd64",
+            "darwin-amd64": "server/dist/plugin-darwin-amd64",
+            "windows-amd64": "server/dist/plugin-windows-amd64.exe"
+        }
+    },
+    "webapp": {
+        "bundle_path": "webapp/dist/main.js"
+    },
+    "settings_schema": {
+        "settings": [
+            {
+                "key": "JitsiURL",
+                "display_name": "Jitsi Server URL",
+                "type": "text",
+                "help_text": "The URL for an on-premise Jitsi server. For example, https://jitsi.example.com.",
+                "placeholder": "https://jitsi.example.com"
+            },
+            {
+                "key": "JitsiEmbedded",
+                "display_name": "Embed Jitsi video inside Mattermost",
+                "type": "bool",
+                "help_text": "(Experimental) When true, Jitsi video is embedded as a floating window inside Mattermost."
+            },
```

Comment:
The ordering seams weird to me. I would expect an experimental setting to be placed at the bottom of the settings list.

Reply:
@jasonblais what do you think?

Reply:
I would like to keep it at the top. It is the most powerful feature of this plugin that we want to highlight to the community. Once it's had enough testing and a UX review, we plan to have it on by default.